src/components/QuizBuilder.ts

```diff
@@ -42,12 +42,12 @@ export class QuizBuilder {
     }
 
     /* get random number for picking a question */
-    randomPick(questions: Question[]) : number{
+    randomPickQuestion(questions: Question[]) : number{
         return Math.floor(Math.random() * questions.length)
     }
 
     /* shuffle function */
-    shuffle(array : any[]) {
+    shuffleQuestionsOrder(array : any[]) {
```

Comment:
This method has zero knowledge about questions and what they are .. it takes _any_ array and shuffles .. which leads to another point:

Is this method really part of a quiz builder?

Reply:
no it doesn't belong. i wanted to use it later for the answers too

Reply:
You could leave it for now and make it a static method .. have a look about static methods (same as in Java)